shells/android/java/arcs/crdt/parcelables/ParcelableCrdtSet.kt

```diff
@@ -0,0 +1,212 @@
+package arcs.crdt.parcelables
+
+import android.os.Parcel
+import android.os.Parcelable
+import arcs.common.Referencable
+import arcs.common.ReferenceId
+import arcs.crdt.CrdtSet
+
+/** Container of [Parcelable] implementations for the data and ops classes of [CrdtSet]. */
+object ParcelableCrdtSet {
+    /** Parcelable variant of [CrdtSet.DataValue]. */
+    data class DataValue(
+        val actual: CrdtSet.DataValue<Referencable>
+    ) : Parcelable {
+        override fun describeContents(): Int = 0
+
+        override fun writeToParcel(parcel: Parcel, flags: Int) {
+            parcel.writeTypedObject(ParcelableVersionMap(actual.versionMap), flags)
+            parcel.writeTypedObject(ParcelableReferencable(actual.value), flags)
+        }
+
+        companion object CREATOR : Parcelable.Creator<DataValue> {
+            override fun createFromParcel(parcel: Parcel): DataValue {
+                val versionMap = parcel.readTypedObject(ParcelableVersionMap.CREATOR)!!.actual
+                val value = requireNotNull(parcel.readReferencable())
+                return DataValue(CrdtSet.DataValue(versionMap, value))
+            }
+
+            override fun newArray(size: Int): Array<DataValue?> = arrayOfNulls(size)
+        }
+    }
+
+    /** Parcelable variant of [CrdtSet.Data]. */
+    data class Data(
+        override val actual: CrdtSet.Data<Referencable>
+    ) : ParcelableCrdtData<CrdtSet.Data<Referencable>> {
+        override var versionMap = actual.versionMap
+
+        override fun writeToParcel(parcel: Parcel, flags: Int) {
+            parcel.writeTypedObject(ParcelableVersionMap(actual.versionMap), flags)
```

Comment:
Another opportunity for `.toParcelable()`

Reply:
Done